### PR TITLE
HP-1063 Fix/popup

### PR DESCRIPTION
**Pull request by mfshao**
Jira Ticket: [HP-1063](https://ctds-planx.atlassian.net/browse/HP-1063)

### Bug Fixes
- Fixed a popup height issue
- Fixed an issue that some urls may have double slash in them

[HP-1063]: https://ctds-planx.atlassian.net/browse/HP-1063?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Popup.less [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 .popup__box {
   width: 70%;
-  height: 100%;
```
Comment (mfshao) [inline src/components/Popup.less:30]: I think this should do the trick @craigrbarnes 🤔

File: src/components/Popup.less [hunk 2]
```diff
@@ -27,7 +27,7 @@
...
+  max-height: 100%;
```
Comment (mfshao) [inline src/components/Popup.less:30]: I think this should do the trick @craigrbarnes 🤔

File: src/actions.js [hunk 3]
```diff
@@ -381,6 +381,7 @@ export const updateSystemUseNotice = (displayUseWarning) => (dispatch) => {
   });
 };
 
+// eslint-disable-next-line max-len
```
[the remark this answers is not in the document]
Comment (mfshao) [reply src/actions.js]: I think the `getState().popups.systemUseWarnPopup` is not being used, so should be ok to remove it to make this even shorter

**Review by jarvisraymond-uchicago — APPROVED** [review]
Looks good to me!

**Pull request merged**